### ISSUE TITLE
[COGNITION]: Text Appointments - Edit buttons would benefit from aria-labels

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2440]
## Description

The `Edit` buttons rely on proximity to a label or heading to answer the question "Edit what?". This can be improved for assistive devices and screen readers by adding `aria-label` attributes to the buttons. Example screenshots attached.

/cc @peggygannon for Content team guidance as needed

## Point of Contact

**VFS Point of Contact:** _Trevor, Peggy_

## Acceptance Criteria

* As a screen reader user, I want to hear contextual aria-labels read aloud. This text could sound like "Edit mobile phone number".
* The text "Edit" should be included in each `aria-label`, ideally as the first text. See the second WCAG guidance link for more on this point.

## Environment

* Windows 10
* Chrome
* JAWS
* https://staging.va.gov/profile

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)
* [Understanding Success Criterion 2.5.3: Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-14 at 3 50 39 PM](https://user-images.githubusercontent.com/934879/66782199-c6d91300-ee9a-11e9-9524-e582c03d8c65.png)

**Comment by lkoenigsberg** [comment]
This will move to the Authenticated Experience board for consideration as we merge account and profile to a single page and redesign.
